client/src/periodUtils.js

```diff
@@ -3,8 +3,6 @@ import PropTypes from "prop-types"
 import React from "react"
 import { momentObj } from "react-moment-proptypes"
 
-const now = moment()
```

Comment:
If the webpage stays open more than a day (like after midnight), global scoped `now` can be misrepresenting the time so i moved it to where it is being used immediately.